include/zim/archive.h

```diff
@@ -140,6 +140,17 @@ namespace zim
        */
       std::vector<std::string> getMetadataKeys() const;
 
+      /** Get the illustration item of the archive.
+       *
+       *  Illustration is a icon for the archive that can be used in catalog and so to illustrate the archive.
+       *
+       *  @param width The width of the illustration to get. Default to 48 (48x48px icon)
+       *  @param height The height of the illustration to get. Default to 48 (48x48px icon)
+       *  @return The illustration item.
+       *  @exception EntryNotFound If no illustration item can be found.
+       */
+      Item getIllustrationItem(unsigned int width=48, unsigned int height=48) const;
```

Comment:
What does this return if the illustration of the exact requested size doesn't exist in the archive? The behaviour for that case must be documented.